lib/graphql/argument.rb

```diff
@@ -65,7 +65,7 @@ def default_value=(new_default_value)
         @default_value = nil
       else
         @has_default_value = true
-        @default_value = new_default_value
+        @default_value = GraphQL::Argument.deep_stringify(new_default_value)
```

Comment:
I had to make this change to fix the tests. The [`order_by` field](https://github.com/rmosolgo/graphql-ruby/blob/c7d537df071b40e2fe9d59ac3a9d8aefb0adb284/spec/support/dummy/schema.rb#L236) in the test schema has a default using a symbol key, which wouldn't get correctly coerced.

Thoughts on the best place to put `deep_stringify`?

Reply:
The move makes sense to me, I don't really have a thought about where to put `deep_stringify`. Since it's a pure function, and private, I think it doesn't matter too much.